Makefile

```
@@ -0,0 +1,32 @@
+TARGET = ZJU_BeamerTemplate
+SRC = $(TARGET).tex
+PDF = $(TARGET).pdf
+
+.PHONY: all clean
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)